datadog_checks_dev/datadog_checks/dev/tooling/commands/clean.py

```diff
@@ -53,12 +66,23 @@ def clean(ctx, check, compiled_only, all_matches, verbose):
             )
     else:
         path = os.getcwd()
+        if path == resolve_path(get_root()):
```

Comment:
Users may use this command a lot just in general so let's add `basepath` to `from ...utils import` and make it

`if basepath(path) in ('integrations-core', 'integrations-extras'):`

Reply:
I'm not sure I understand the difference between these two? But, I agree that it's clearer